### ISSUE TITLE
Half bubble show on screen end.

### ISSUE THREAD

**Issue by SohailZahidGit** [SamThompson/BubbleActions#27]
If we add this on recycleview and user preform long press on edge on item bubble will show partially.
![20180125_154654](https://user-images.githubusercontent.com/13638764/35384538-69d63a80-01e7-11e8-91a7-489c3367429b.png)

**Comment by SamThompson** [comment]
Does it exhibit the same behavior when you long press on the right edge? If so, the angle it offsets probably just needs to be tweaked.

**Comment by SohailZahidGit** [comment]
@SamThompson yes `it exhibit the same behavior when you long press on the right edge` but i tweaked it and its working fine for me.
